build_scripts/assets/rpm/postinst.sh

```diff
@@ -3,5 +3,5 @@
 
 set -e
 
-ln -s /usr/lib/chia-blockchain/resources/app.asar.unpacked/daemon/chia /usr/bin/chia || true
-ln -s /usr/lib/chia-blockchain/resources/app.asar.unpacked/daemon /opt/chia || true
+ln -s /opt/chia/resources/app.asar.unpacked/daemon/chia /usr/bin/chia || true
+ln -s /opt/chia/resources/app.asar.unpacked/daemon /opt/chia || true
```

Comment:
Yeah agree with Will, since there is already a directory here now. The more important one is the link to `/usr/bin/chia` - the link for /opt/chia was to make it easier to find chia in both the UI and CLI only installers

Reply:
I've removed the line.